ibis/backends/flink/tests/test_ddl.py

```diff
@@ -94,15 +113,56 @@ def test_create_table(
 
 
 def test_create_source_table_with_watermark(
-    con, functiona_alltypes_schema, temp_table, csv_connector_configs
+    con, functiona_alltypes_schema, temp_table, csv_source_configs
 ):
     new_table = con.create_table(
         temp_table,
         schema=functiona_alltypes_schema,
-        tbl_properties=csv_connector_configs("functional_alltypes"),
+        tbl_properties=csv_source_configs("functional_alltypes"),
         watermark=ibis.watermark(
             time_col="timestamp_col", allowed_delay=ibis.interval(seconds=15)
         ),
     )
     assert temp_table in con.list_tables()
     assert new_table.schema() == functiona_alltypes_schema
+
+
+def test_insert_list_values_into_table(con, tempdir_sink_configs):
+    sink_schema = sch.Schema({"name": dt.string, "age": dt.int64, "gpa": dt.float64})
+    with tempfile.TemporaryDirectory() as tempdir:
+        con.create_table(
+            "tempdir_sink",
+            schema=sink_schema,
+            tbl_properties=tempdir_sink_configs(tempdir),
+        )
+        con.insert(
+            "tempdir_sink",
+            [("fred flintstone", 35, 1.28), ("barney rubble", 32, 2.32)],
```

Comment:
I think this should be parametrized to also cover inserting DataFrames and pyarrow tables.